lib/chewy/index.rb

```diff
@@ -146,6 +146,8 @@ def prefix
       # passed.
       #
       def define_type(target, options = {}, &block)
+        raise 'Multiple types are deprecated' if type_hash.present?
```

Comment:
Instead of raise if type_hash is present, we could raise when a user is trying to set more than one type.
```suggestion
        raise "Multiple types aren't supported by ElasticSearch v6" if type_hash.size > 1
```
I suggest this or it will fail in the projects.

Reply:
`if type_hash.present?` guard will be triggered on the 2nd invocation of `define_type` in some particular index class. While `if type_hash.size > 1` will allow to declare 2 types in 1 index.